### PR TITLE
Personal/sowu/version validation readme

### PR DESCRIPTION
**Pull request by sowu880**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -242,17 +242,18 @@ The command-line tool can be used to anonymize a folder containing FHIR resource
...
+> .\Fhir.Anonymizer.<version>.CommandLineTool.exe -i myInputFolder -o myOutputFolder
 ```
 
 ## Configuration file format
 
-The configuration is specified in JSON format. It has two high-level sections. One of these sections, namely _fhirPathRules_ is meant to specify de-identification methods for data elements. The second section named _parameters_ affects global behavior. _fhirPathRules_ are executed in the order of appearance in the configuration file. The content of _fhirPathRules_ is version dependent since resources are different in R4 and STU 3. 
+The configuration is specified in JSON format. It has three high-level sections. One of these sections, namely _fhirVersion_ specify the configuration file's version for anonymizer. The second section named _fhirPathRules_ is meant to specify de-identification methods for data elements. The third section named _parameters_ affects global behavior. _fhirPathRules_ are executed in the order of appearance in the configuration file. 
 
-Here is a sample configuration:
+Here is a sample configuration for R4:
```
Comment (moria97) [inline README.md:252]: We can give a stu3 sample as well

File: README.md [hunk 2]
```diff
@@ -269,7 +270,15 @@ Here is a sample configuration:
...
 ```
+### FhirVersion Specify
+| fhirVersion | Desciption |
+| ----- | ----- |
+|Stu3|Specify STU 3 version for the configuration file|
+|R4|Specify R4 version for the configuration file|
+|Empty or Null| The configuration file will be specified the same version with the running executable file.
 
+> **[!NOTE]**
+> When  _fhirVersion_ section is missing or empty, the program will not throw an exception but give a warning information.
```
Comment (moria97) [inline README.md:281]: Seems this section is not necessary as we have explained in line 278

File: README.md [hunk 3]
```diff
@@ -269,7 +270,15 @@ Here is a sample configuration:
   }
 }
 ```
+### FhirVersion Specify
```
Comment (moria97) [inline README.md:273]: Specification?

File: README.md [hunk 4]
```diff
@@ -269,7 +270,15 @@ Here is a sample configuration:
   }
 }
 ```
+### FhirVersion Specify
+| fhirVersion | Desciption |
+| ----- | ----- |
+|Stu3|Specify STU 3 version for the configuration file|
+|R4|Specify R4 version for the configuration file|
+|Empty or Null| The configuration file will be specified the same version with the running executable file.
```
Comment (moria97) [inline README.md:278]: Add another line for other values?  Other values will raise an **exception.

File: README.md (hunk 2 above)
Comment (sowu880) [reply README.md:281]: But we haven't explain the warning information in the table.

File: README.md (hunk 1 above)
Comment (sowu880) [reply README.md:252]: Seems redundant?

File: README.md (hunk 4 above)
Comment (sowu880) [reply README.md:278]: Thanks , added a new line in table.

File: README.md (hunk 2 above)
Comment (moria97) [reply README.md:281]: For customers, this warning is not worth emphasizing here.

Comment (BoyaWu10) [reply README.md:281]: Is the warning always printed (both with -v and without -v)?

File: README.md (hunk 4 above)
Comment (BoyaWu10) [reply README.md:278]: Is the new line already pushed into this PR? It seems I cannot find it.

File: README.md (hunk 3 above)
Comment (BoyaWu10) [reply README.md:273]: +1

File: README.md (hunk 2 above)
Comment (sowu880) [reply README.md:282]: always.

File: README.md (hunk 4 above)
Comment (ranvijaykumar) [reply README.md:278]: For empty or null, suggest changing the working to : "The configuration file targets the same FHIR version as the executable."

Comment (sowu880) [reply README.md]: Thanks, updated here.

**Pull request merged**